### PR TITLE
Fix demo queries

### PR DESCRIPTION
**Pull request by dknochen**

### CODE REVIEW

File: queries/modules/io/28/apps/request.jq [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+         { "Default" : null }
+             [$all and (not $has-default) and exists($type)],
+         { "Default" : "xbrl28:Domain" }
+             [$all and (not $has-default) and empty($type)]
+       |}
+    }
+  return
+    {|
+      $request,
+      { dimensions: [ $dimensions ] }
```
Comment (gislenius) [inline queries/modules/io/28/apps/request.jq]: Why not {| $dimensions |} ?

File: queries/modules/io/28/apps/request.jq [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+};
+
+declare function request:hypercube(
+  $request as object,
+  $entities as object*) as object
+{
+  let $open as boolean := $request.open
+  let $hypercube-spec :=
+  {|
+    for $parameter in request:param-names($request)
```
Comment (gislenius) [inline queries/modules/io/28/apps/request.jq]: That seems to be duplicate code. Any way we can reuse use request().dimensions?

**Comment by gislenius** [comment]
Back to you Dennis ;-) Nice refactoring. I like the new request module.

**Comment by dknochen** [comment]
@gislenius addressed your comments. Thanks a lot :)

**Pull request merged**